pyanaconda/ui/gui/spokes/lib/resize.py

```diff
@@ -68,6 +68,13 @@ def __init__(self, data, storage, payload):
         self.storage = storage
         self.payload = payload
 
+        self._storage_proxy = STORAGE.get_proxy()
+
+        # Get the required device size.
+        required_space = self.payload.space_required.get_bytes()
+        required_size = self._storage_proxy.GetRequiredDeviceSize(required_space)
+        self._required_size = Size(required_size)
```

Comment:
Not sure if this is the correct way, what if we would want to re-use the dialog? 

I would rather see the original variant.

Reply:
We always create a new dialog. I am not even sure if a dialog can be run twice.

Reply:
Btw. we set a label with this value in the `__init__` method, so the original variant is pointless.

Reply:
In that case this makes the think more obvious. Thanks!